pkgs/top-level/all-packages.nix

```diff
@@ -9739,6 +9739,12 @@ with pkgs;
 
   libcCross = assert stdenv.targetPlatform != stdenv.buildPlatform; libcCrossChooser stdenv.targetPlatform.libc;
 
+  ldd = {
```

Comment:
Wouldn't it just be easier to set this to `stdenv.hostPlatform.stdenv.cc.libc`? All libcs should provide ldd, if I'm not mistaken.

Reply:
Yeah that sounds like a good idea.